test/test_transforms.py

```diff
@@ -130,6 +131,22 @@ def test_ten_crop(self):
                 assert len(results) == 10
                 assert expected_output == results
 
+    def test_randomresized_params(self):
+        height = random.randint(24, 32) * 2
+        width = random.randint(24, 32) * 2
+        img = torch.ones(3, height, width)
+        to_pil_image = transforms.ToPILImage()
+        img = to_pil_image(img)
+        size = 100
+        epsilon = 0.01
+        aspect_ratio_range = (0.75, 1.3)
+        scale_range = (0.08, 1)
+        for i in range(10):
+            randresizecrop = transforms.RandomResizedCrop(size, scale_range, aspect_ratio_range)
+            _, _, h, w = randresizecrop.get_params(img, scale_range, aspect_ratio_range)
+            aspect_ratio_obtained = w / h
+            assert min(aspect_ratio_range) - epsilon <= aspect_ratio_obtained <= max(aspect_ratio_range) + epsilon
```

Comment:
This doesn't help because the problem isn't round-off error.
For example, when aspect_ratio_range = (0.75, 0.75) (mentioned in #618 ), you still get 50% chance error.
The root cause is here:
https://github.com/pytorch/vision/pull/715/files#r248648385

Reply:
@kouyoumin  yeah you are right !! 
Thanks